app/src/pages/inside/dashboardPage/modals/addEditModal/addEditModal.jsx

```diff
@@ -402,42 +337,15 @@ export class AddEditModal extends Component {
               label={intl.formatMessage(messages.configurationTitle)}
               labelWidth={LABEL_WIDTH}
             >
-              <div className={cx('configuration-content')}>
-                {!pastedConfig ? (
-                  <>
-                    <GhostButton icon={PasteIcon} onClick={this.handlePasteConfiguration}>
-                      {intl.formatMessage(messages.pasteConfiguration)}
-                    </GhostButton>
-                    <div className={cx('configuration-hint')}>
-                      {intl.formatMessage(messages.pasteConfigHint)}
-                    </div>
-                  </>
-                ) : (
-                  <>
-                    <div className={cx('configuration-pasted')}>
-                      <span className={cx('pasted-icon')}>{Parser(StorageIcon)}</span>
-                      <span className={cx('pasted-text')}>
-                        {intl.formatMessage(messages.configurationPasted)}
-                      </span>
-                    </div>
-                    <div className={cx('configuration-hint')}>
-                      {intl.formatMessage(messages.canAdjustNameDesc)}
-                    </div>
-                    <div className={cx('remove-configuration')}>
-                      <GhostButton
-                        icon={RemovePastedIcon}
-                        onClick={this.handleRemoveConfiguration}
-                        className={cx('remove-button')}
-                        color="red"
-                      >
-                        {intl.formatMessage(messages.removeConfiguration)}
-                      </GhostButton>
-                      <div className={cx('configuration-hint')}>
-                        {intl.formatMessage(messages.removeConfigurationHint)}
-                      </div>
-                    </div>
-                  </>
-                )}
+              <FieldProvider
+                name="configuration"
+                maxLength="INFINITY"
+                placeholder={intl.formatMessage(messages.pasteConfiguration)}
+              >
+                <InputTextArea />
+              </FieldProvider>
```

Comment:
I see the error state on designs for this field.
Could you please add validation for valid JSON input and corresponding error message?

Reply:
Not valid anymore.